src/components/Pricing/constants.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Cohorts, FeatureFlags, Funnels, PathAnalysis, SessionRecordings } from 'components/Icons/Icons'
+export const SCALE_MINIMUM_PRICING = 1_000
+export const SCALE_MINIMUM_EVENTS = 4_444_444
```

Comment:
Is there a reason not to set this at 1,000,000? Presuming that if a user has more than 1M events, their choice between Cloud and Scale may be dictated by hosting/security needs and not cost... Open-ended question.

Also, for what it's worth I have been confused by the pricing calculator on [license.posthog.com](https://license.posthog.com/) — it reads as if there's a discontinuity between <= 8,888,888 events ($2000) and >= 10,000,000 events ($0.000225/ea). This makes it unclear what the unit cost would be between SCALE_MINIMUM_EVENTS and 10,000,000.

I know the pricing code here does _not_ say that, but the language seems to suggest it. Happy to put up a PR on license.posthog with a wording suggestion unless you have something else in mind.

Reply:
> Is there a reason not to set this at 1,000,000? Presuming that if a user has more than 1M events, their choice between Cloud and Scale may be dictated by hosting/security needs and not cost... Open-ended question.

4,444,444*0.000225=$1000/month, hence starting the calculator at that point?

> Also, for what it's worth I have been confused by the pricing calculator on license.posthog.com — it reads as if there's a discontinuity between <= 8,888,888 events ($2000) and >= 10,000,000 events ($0.000225/ea). This makes it unclear what the unit cost would be between SCALE_MINIMUM_EVENTS and 10,000,000.

Yep this is a good point, I'll have a look at this.